Assets/Scripts/Models/Buildable/Components/BuildableComponent.cs

```diff
@@ -191,7 +191,7 @@ protected bool AreParameterConditionsFulfilled(List<ParameterCondition> conditio
         {
             componentTypes = new Dictionary<string, System.Type>();
 
-            foreach (Assembly assembly in AppDomain.CurrentDomain.GetAssemblies())
+            foreach (Assembly assembly in AppDomain.CurrentDomain.GetAssemblies().Where(asm => !CSharpFunctions.IsDynamic(asm)))
```

Comment:
in BuildableComponent, ln.12 I see:
```cs
using System.Linq;
```
also, it was compiled several times. I've removed unused using from the file to trigger new build.